tests/PHPStan/Parser/TypeParserTest.php

```diff
@@ -988,11 +988,11 @@ public function provideParseData(): array
 						),
 					]),
 					[
-						new CallableTypeTemplateNode(new IdentifierTypeNode('Tx'), new UnionTypeNode([
+						new TemplateTagValueNode('Tx', new UnionTypeNode([
 							new IdentifierTypeNode('X'),
 							new IdentifierTypeNode('Z'),
-						])),
-						new CallableTypeTemplateNode(new IdentifierTypeNode('Ty'), new IdentifierTypeNode('Y')),
+						]), ''),
```

Comment:
https://github.com/phpstan/phpdoc-parser/blob/1.25.0/src/Ast/PhpDoc/TemplateTagValueNode.php#L26 `description` should be made optional